src/openrct2-ui/interface/ViewportInteraction.cpp

```diff
@@ -697,6 +697,34 @@ static void ViewportInteractionRemoveLargeScenery(TileElement* tileElement, cons
     }
 }
 
+struct PeepDistance
+{
+    Peep* peep = nullptr;
+    int32_t distance = std::numeric_limits<int32_t>::max();
```

Comment:
Seems like a good place to use `decltype` instead of having the type there twice:

```suggestion
    int32_t distance = std::numeric_limits<decltype(distance)>::max();
```